style.css

```diff
@@ -32,6 +33,42 @@ body {
   position: relative;
 }
 
+/* start mobile-menu */
+.mob-menu-close-icon {
+  color: #fff;
+  align-self: flex-end;
+  padding: 0;
+}
+
+.mob-menu--hide {
+  display: none;
+  width: 0;
+  height: 0;
+}
+
+.mobile-menu--show {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 1em 0.75em;
+  width: 100vw;
+  height: 100vh;
+  color: #fff;
+  background: rgba(96, 112, 255, 0.9);
+  font-family: "Poppins", sans-serif;
+  font-weight: 600;
+  font-size: 2rem;
+  line-height: 44px;
+  backdrop-filter: blur(4px);
+}
```

Comment:
- [ ] **[OPTIONAL]** You've done a fantastic job with your mobile menu! I have a small optional suggestion that could enhance **its professional appearance.** Would you consider adding a `position fixed` to the overlay when the menu is opened? Currently, users **are able to scroll down while the mobile menu remains at the top,** and implementing this change would address that issue. 
  
    [scrnli_6_28_2023_7-07-14 AM.webm](https://github.com/the-faizmohammad/my-portfolio/assets/107952724/0fc74349-6ace-4fa5-9978-9ffdc94ef5d5)